src/less-watch-compiler.js

```diff
@@ -93,7 +95,10 @@ function init(){
       });
   }
   
-  console.log('Watching directory for file changes.');
+  if (lessWatchCompilerUtils.config.runonce === true)
+    console.log('Running less-watch-compiler once.');
```

Comment:
consider wrap if/else block in {} brackets

Reply:
- [ ] Wrap if/else in {}